code/modules/holiday/holidays.dm

```diff
@@ -659,6 +659,9 @@
 	drone_hat = /obj/item/clothing/head/costume/santa
 	mail_holiday = TRUE
 
+/datum/holiday/xmas/getStationPrefix()
+	return pick("Jesus", "Yule", "Bible", "Elf", "Santa", "Sleigh", "Fir", "Chimney", "Sack", "Present", "Crucifixion", "Birthday", "Claus", "Merry", "Ho Ho Ho", "Jolly")
```

Comment:
```suggestion
	return pick(
		"Jesus",
		"Yule",
		"Bible",
		"Elf",
		"Santa",
		"Sleigh",
		"Fir",
		"Chimney",
		"Sack", 
		"Present",
		"Crucifixion",
		"Birthday",
		"Claus",
		"Merry",
		"Ho Ho Ho",
		"Jolly".
	)
```

then can you alphabetize this list? there's a few extensions for vscode that'll do it for you. i realize that multilining this list may make it look weird, but i think we should really be splitting the holidays up into their own files in a dedicated folder and doing this anyways because of how much it helps maintainability.